src/utils/xrAI/factory_api.h

```diff
@@ -3,12 +3,12 @@
 #include "xrServerEntities/xrServer_Object_Base.h"
 
 extern "C" {
-typedef XR_IMPORT IServerEntity* __stdcall Factory_Create(LPCSTR section);
-typedef XR_IMPORT void __stdcall Factory_Destroy(IServerEntity*&);
+using CreateEntity = XR_IMPORT IServerEntity* __stdcall(LPCSTR section);
+using DestroyEntity = XR_IMPORT void __stdcall(IServerEntity*&);
```

Comment:
This is unnecessary and not related to this PR. Also, probably in future, xrSE_Factory will be refactored, so, this change can cause conflicts or just unnecessary changes.

Reply:
Actually related. After addind xrEngine in the scope you get collision for Factory_Create/Destroy names. This is a fix for that.

Reply:
This time is my turn to be AFK for some time so please manage this PR by yourself or drop it in case of better solution. It was just a quick fix

Reply:
Oh.. I didn't think about it. Ok, then!

Reply:
> This time is my turn to be AFK for some time so please manage this PR by yourself or drop it in case of better solution. It was just a quick fix

Ok! Just renamed second commit to somewhat better name.